pkg/lib/v0_1_3/remediations.go

```diff
@@ -0,0 +1,35 @@
+package v0_1_3
```

Comment:
Does it make sense to create a new package "v0_1_3" for this? 
In addition, we have to change this every time we release a new spec (and in fact, spec 0.1.3 does not support this type of remediation, or does it?

Reply:
The idea of this package structure is to have easy and stable names for types.
Before, we had the type `Remediation` and `RemediationV02`. Here, it is unclear which spec version the types belong to.

Therefore, I introduced a new package named `v0_1_3`, which should contain all _new_ types compliant to spec version v0.1.3.
This is very similar as it is handled in k8s, istio, etc. 
For example, see https://github.com/istio/api/blob/master/networking/v1alpha3/virtual_service.pb.go

Reply:
Okay, thanks for bringing this up!

Then I vote that we call it `v0_1_4` (assuming our next spec version should be 0.1.4 - let's just set that into stone, shall we?) and start following that principle :+1:

Reply:
You are right, it must be `v0_1_4`.
Thank you!